### DISCUSSION TITLE
[Feq Req]: Supports sending metrics about itself to InfluxDB

### DISCUSSION THREAD

**Discussion by deepujain** [grafana/grafana#36976] [General]
With 3.1 Grafana now supports sending metrics about itself.

Can grafana send metrics about itself to InfluxDB ?

**Comment by smeapng** [comment]
+1

**Comment by J-Mx** [comment]
+1

**Comment by karthikeyan95** [comment]
is this possible with elasticsearch as data source.
can grafana send metrics data to elasticsearch or to some other application. 
if it is possible, can anyone suggest any document or method to do so.

**Comment by deepujain** [comment]
+1 for influx

On Fri, Aug 19, 2016 at 6:35 AM, karthikeyan95 notifications@github.com
wrote:

> is this possible with elasticsearch as data source.
> can grafana send metrics to elasticsearch or to some other application.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> https://github.com/grafana/grafana/issues/5596#issuecomment-241019912,
> or mute the thread
> https://github.com/notifications/unsubscribe-auth/AAY0-bbAFuWYnO4XuTWConrPszEOUy5Uks5qhbE6gaJpZM4JOBWp
> .

## 

Deepak

**Comment by bergquist** [comment]
This is not something the Grafana team will be working on for a foreseeable future. 

If anyone wants to submit a PR I would be happy to review it. This should be implemented as a new "bridge"  just like the graphite bridge  in https://github.com/grafana/grafana/tree/master/pkg/metrics

**Comment by Hipska** [comment]
Hi, is there any update on this? Would Grafana team consider working on this now?

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #83020. Feel free to continue the discussion around this there. Thank you!
